helm/bigmon/charts/main/templates/configmap.yaml

```diff
@@ -16,54 +16,6 @@ kind: ConfigMap
 metadata:
```

Comment:
You can delete the env stuff completely from configmap.yaml and deployment.yaml as
proxies are set at

https://github.com/PanDAWMS/panda-k8s/blob/main/secrets/templates/bigmon.yaml#L66

Reply:
You need to remove

https://github.com/PanDAWMS/panda-k8s/blob/main/helm/bigmon/charts/main/templates/deployment.yaml#L101-L102

BTW, please submit pull requests once you test your changes on your folks. The master branch should be functional.

Reply:
Oh yes, I was looking for the proxy setting. Yes I have converted it into WIP.